packages/dev/core/src/Misc/environmentTextureTools.ts

```diff
@@ -360,6 +365,7 @@ export async function CreateEnvTextureAsync(texture: BaseTexture, options: Creat
         info.irradiance.irradianceTexture = {
             size: irradianceTexture.getSize().width,
             faces: [],
+            dominantDirection: texture._dominantDirection?.asArray() || [0, 0, 1],

```

Comment:
I think we should leave it undefined if not there so that we do not end up with a wrong direction at run time

Reply:
Done